cernopendata_client/downloader.py

```diff
@@ -28,6 +29,26 @@
 
 from .validator import validate_range
 from .printer import display_message
+from .verifier import get_file_checksum
+
+
+def check_error(path, file_location, protocol, retry):
+    """Check downloaded file for error page."""
+    file_name = file_location.split("/")[-1]
+    file_path = path + "/" + file_name
+    error_page = (
+        os.path.getsize(file_path) == 3846
+        and get_file_checksum(file_path) == "adler32:a82d5324"
+    )
+    if error_page:
+        display_message(msg_type="note", msg="Retrying")
+        if retry <= 1:
+            display_message(msg_type="error", msg="No. of retries exceeded.")
```

Comment:
I have tried this and I'm seeing:

```console
==> Downloading file 1 of 1
  -> File: ./3005/0d0714743f0204ed3c0144941e6ce248.configFile.py
  -> Progress: 3/3 kiB (100%)
==> Verifying file 0d0714743f0204ed3c0144941e6ce248.configFile.py...
  -> Expected size 3644, found 3644
  -> Expected checksum adler32:be83a186, found adler32:be83a186
==> Success!
==> Downloading file 1 of 1
  -> File: ./3005/0d0714743f0204ed3c0144941e6ce248.configFile.py
  -> Progress: 3/3 kiB (100%)
  -> Retrying
  -> File: ./3005/0d0714743f0204ed3c0144941e6ce248.configFile.py
  -> Progress: 3/3 kiB (100%)
  -> Retrying
  -> File: ./3005/0d0714743f0204ed3c0144941e6ce248.configFile.py
  -> Progress: 3/3 kiB (100%)
  -> Retrying
  -> File: ./3005/0d0714743f0204ed3c0144941e6ce248.configFile.py
  -> Progress: 3/3 kiB (100%)
  -> Retrying
  -> File: ./3005/0d0714743f0204ed3c0144941e6ce248.configFile.py
  -> Progress: 3/3 kiB (100%)
  -> Retrying
  -> File: ./3005/0d0714743f0204ed3c0144941e6ce248.configFile.py
  -> Progress: 3/3 kiB (100%)
  -> Retrying
  -> File: ./3005/0d0714743f0204ed3c0144941e6ce248.configFile.py
  -> Progress: 3/3 kiB (100%)
  -> Retrying
  -> File: ./3005/0d0714743f0204ed3c0144941e6ce248.configFile.py
  -> Progress: 3/3 kiB (100%)
  -> Retrying
  -> File: ./3005/0d0714743f0204ed3c0144941e6ce248.configFile.py
  -> Progress: 3/3 kiB (100%)
  -> Retrying
  -> File: ./3005/0d0714743f0204ed3c0144941e6ce248.configFile.py
  -> Progress: 3/3 kiB (100%)
  -> Retrying
==> ERROR: No. of retries exceeded.
==> Downloading file 1 of 1
  -> File: ./3005/0d0714743f0204ed3c0144941e6ce248.configFile.py
  -> Progress: 3/3 kiB (100%)
  -> Retrying
  -> File: ./3005/0d0714743f0204ed3c0144941e6ce248.configFile.py
  -> Progress: 3/3 kiB (100%)
  -> Retrying
  -> File: ./3005/0d0714743f0204ed3c0144941e6ce248.configFile.py
  -> Progress: 3/3 kiB (100%)
  -> Retrying
  -> File: ./3005/0d0714743f0204ed3c0144941e6ce248.configFile.py
  -> Progress: 3/3 kiB (100%)
==> Verifying file 0d0714743f0204ed3c0144941e6ce248.configFile.py...
  -> Expected size 3644, found 3644
  -> Expected checksum adler32:be83a186, found adler32:be83a186
==> Success!
```

Several observations:

- The order of output messages seems weird? Out of order?
- The outuput message cosmetics: "Retrying (3/10)..." specifying the countdown.
- The error message cosmetics: ERROR: Number of retries exceeded. 
- Perhaps we should have some micro-sleep time?  The errors were succeeding very fast for me... It may help if retries are happening after say 2 seconds.  (This time could be made configurable in `config.py`.)